nixos/modules/profiles/qemu-guest.nix

```diff
@@ -15,5 +15,5 @@
       hwclock -s
     '';
 
-  security.rngd.enable = lib.mkDefault false;
```

Comment:
why not just drop this setting?

Reply:
Sorry, that's what I had intended, instead I accidentally just reverted c27360ae47a76fe8de6a23a21e19e3d85e13dc0c.
Fixed.